script/release-notes/notes.js

```diff
@@ -521,10 +521,12 @@ const getNotes = async (fromRef, toRef, newVersion) => {
       }
 
       // try to pull a release note from the pull comment
-      const prParsed = {}
-      parseCommitMessage(`${prData.data.title}\n\n${prData.data.body}`, pr.owner, pr.repo, prParsed)
+      const prParsed = parseCommitMessage(`${prData.data.title}\n\n${prData.data.body}`, pr.owner, pr.repo)
       commit.note = commit.note || prParsed.note
       commit.type = commit.type || prParsed.type
+      if (prParsed.type === 'breaking-change') {
+        commit.type = prParsed.type
+      }
```

Comment:
I the flow of "when does `commit.type` get set" is a little hard to read with this and 526 side-by-side.

What we joined them this way:

```js
if (!commit.type || prParsed.type === 'breaking-change') {
  commit.type= prParsed.type
} 
```

Reply:
yeah, makes sense. Did this & changed the `note` logic to match.